jobs/redis-blacksmith-plans/templates/plans/cluster-6/manifest.yml

```diff
@@ -8,6 +8,20 @@ meta:
   default:
     azs: [z1, z2]
 
+variables:
+- name: redis_cluster_crt
+  type: certificate
+  options:
+    ca: (( concat "/" meta.bosh_env "-bosh/" meta.environment "-blacksmith/blacksmith_services_ca" ))
+    common_name: (( concat meta.environment "-redis-cluster.bosh" ))
+  consumes:
+    alternative_name: 
+      from: redis-san 
+      properties: { wildcard: true }
+
+features:
+  use_dns_addresses: true
```

Comment:
@itsouvalas is this required for Redis too or can I remove this feature?

Reply:
see comment bellow